src/adapters/assignprivilegelocater.ts

```
@@ -8,4 +9,11 @@ export interface IServicelocatorprivilegeRole {
     response: Response
   );
   getPrivilegeRole(userId, request, response: Response);
+  getGroupedPermissionsForRole(roleId: string, response: Response);
+  deletePrivilegeFromRole(roleId: string, privilegeId: string, response: Response);
```

ESLint

[error] Replace `roleId:·string,·privilegeId:·string,·response:·Response` with `⏎····roleId:·string,⏎····privilegeId:·string,⏎····response:·Response⏎··`

(prettier/prettier)